catroid/res/values-de/strings.xml

```diff
@@ -36,6 +35,7 @@
   <string name="unpacking">Auspacken</string>
   <string name="unpack">Pack es aus</string>
   <string name="copy_addition">Kopie</string>
+  <!--<string name="unpacking_addition">entpackt</string>-->
```

Comment:
please remove strings that are no longer needed instead of commenting. if you remove anything from the values/strings.xml you'll have to remove it within all other language files too (Android Studio provides the functionality). When you add strings you don't need to change anything. When you change strings (their identifiers not their content), be sure to remove them first and add the new string only in values/strings.xml